projects/plugins/social/src/class-jetpack-social.php

```diff
@@ -307,6 +311,10 @@ class_exists( 'Jetpack' ) ||
 			)
 		);
 
+		if ( 'draft' === get_post_status() && self::can_use_analytics() && ! self::is_review_request_dismissed() ) {
+			wp_add_inline_script( 'jetpack-social-editor', Connection_Initial_State::render(), 'before' );
```

Comment:
I see the following error in the editor at the moment, as soon as I go to Posts > Add New.

```
editor.js?minify=false&ver=1bed4da197b244cd9534:2
Jetpack Connection package: Initial state is missing. Check documentation to see how to use the Connection composer package to set up the initial state.
```

Maybe we should always pass that data to the editor? I would assume we will use it for other elements in the future.

Reply:
Just noticed this myself :)